lib/raygun/raygun.rb

```diff
@@ -208,6 +212,35 @@ def print_next_steps_for_custom_repo
 
     protected
 
+    def fetch_rubocop_file
+      sha = shell("git ls-remote https://github.com/#{C5_CONVENTIONS_REPO} master")
+      sha = sha.slice(0..6)
+
+      rubocop_file = "https://raw.githubusercontent.com/#{C5_CONVENTIONS_REPO}/master/rubocop/rubocop.yml"
+      begin
+        prefix = [
+          "# Sourced from #{C5_CONVENTIONS_REPO} @ #{sha}",
+          '#',
+          '# If you make changes to this file, consider opening a PR to backport them to the c5-conventions repo:',
+          "# https://github.com/#{C5_CONVENTIONS_REPO}/blob/master/rubocop/rubocop.yml",
+          '',
+          ''
+        ].join("\n")
+        rubocop_contents = URI.open(rubocop_file)
+        write_file("#{@app_dir}/.rubocop.yml", prefix + rubocop_contents.string)
+      rescue Errno::ENOENT, OpenURI::HTTPError => e
+        puts ''
+        puts "Failed to find the CarbonFive conventions rubocop file at #{rubocop_file}".colorize(:light_red)
+        puts "You'll have to manage you're own `.rubocop.yml` setup".colorize(:light_red)
+      end
+    end
+
+    def write_file(filename, contents)
+      fp = File.open(filename, 'w')
+      fp.write(contents)
+      fp.close
+    end
```

Comment:
I think this is equivalent to Ruby's:

```ruby
IO.write(filename, contents)
```

https://ruby-doc.org/core-2.7.0/IO.html#method-c-write